plsql.pls

```diff
@@ -0,0 +1,4 @@
+BEGIN
+dbms_output.put_line ('Hello World');
+END;
+/
```

Comment:
From my testings, it seems like this line causes an error. Removing it fixes this, but I have no experience with PL/SQL. Can you confirm?

Reply:
https://www.thecrazyprogrammer.com/2017/09/plsql-online-compiler.html see this link to compile this plsql program. I think after following these steps you will not get any error.

Reply:
Ahh thanks for the link! It works well in there.